l10n_br_account/models/res_config.py

```diff
@@ -10,6 +10,10 @@ class ResConfig(models.Model):
     _inherit = 'account.config.settings'
 
```

Comment:
@Hendrixcosta could you please change ResConfig(models.Model) to ResConfig(models.TransientModel) ?
indeed it's a temporary object https://github.com/odoo/odoo/blob/8.0/addons/account/res_config.py#L32 and in the new API it gives us a models.TransientModel